spec/acceptance/api/internal/services_api_spec.rb

```diff
@@ -55,7 +55,7 @@
         example_request 'Get Service by ID' do
           expect(status).to eq 200
           expect(response_json['service']['id']).to eq id
-          expect(response_json['service']['state']).to eq 'suspended'
+          expect(response_json['service']['state']).to eq 'active'
```

Comment:
minor nit: the title is inconsistent with the other commit - spec/services_api vs service_api_spec